valora-inc/wallet
# chore: restore merchant payments feature

### Description

Restores the Merchant Payment feature removed in #3190.

### Test plan

Tested on Android using https://sandbox-celo-demo-store.demo.firstdag.com. All steps except the final step were successful - this is likely an issue with the sandbox site.

1. Select an item.
2. Scan with QR code scanner in Valora.
3. Send Payment.
4. Observe payment sent and balance updated.
5. Observe payment on website is successful.

#### Video

https://user-images.githubusercontent.com/26950305/214186864-f4432e2c-769f-49bd-ac56-c95187c2f022.mov



### Related issues

- Fixes RET-564

### Backwards compatibility

Yes
